### PR TITLE
include installation and run instructions in README

### PR DESCRIPTION
**Pull request by robskrob**
@chadwpry I updated the README. Feel free to suggest changes or ask me to add information if it is missing.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,25 +1,97 @@
...
-=============
+### Getting Started
 
-$ sudo npm start
+Follow the steps below to setup a development environment. This project
+expects you will be using a flavor of Unix that supports a standard node.js
+environment. The project has been tested in an OS X 10.11.x environment.
 
-# extension
+_*** the project has not been tested in a windows environment_
```
Comment (chadwpry) [inline README.md]: You can remove this note

File: README.md [hunk 2]
```diff
@@ -1,25 +1,97 @@
...
+The project will allow users to capture data on pages and store it to catalogue and organize for later use.
 
-Run
-=============
+### Getting Started
 
-$ sudo npm start
+Follow the steps below to setup a development environment. This project
+expects you will be using a flavor of Unix that supports a standard node.js
+environment. The project has been tested in an OS X 10.11.x environment.
```
Comment (chadwpry) [inline README.md]: You can leave out the OS X environment that the project has been tested against and just note unix.

File: README.md [hunk 3]
```diff
@@ -1,25 +1,97 @@
...
-$ npm run build
+    `$ export NODE_ENV=development`
+
+### API
+
+3. Install node packages
+
+    `npm install`
+
+4. Create a SSL certificate
```
Comment (chadwpry) [inline README.md]: Let's note this is a "self signed"  SSL certificate.

File: README.md [hunk 4]
```diff
@@ -1,25 +1,97 @@
...
+
+    `openssl req -nodes -x509 -newkey rsa:2048 -keyout credentials/key.pem -out credentials/cert.pem -days 365`
+
+5. Start server
+
+    `npm start`
+
+### Extension
```
Comment (chadwpry) [inline README.md]: Let's move the extension information into a separate README inside the extension folder.

File: README.md [hunk 5]
```diff
@@ -1,25 +1,97 @@
...
+    `git clone git@github.com:chadwpry/prototype-content-consumer.git`
 
-Run
-=============
+2. Set NODE_ENV variable to development. It can be included in a .bash_profile or .bashrc configuration.
 
-$ npm run build
+    `$ export NODE_ENV=development`
+
+### API
```
Comment (chadwpry) [inline README.md]: Let's keep the API and Extension instructions separate. Move the details for the Extension into a README inside the extension folder.

File: README.md [hunk 6]
```diff
@@ -1,25 +1,97 @@
...
+        }
+     }
+
+#### Extension
+
+1. Go to chrome://extensions/ in your browser and click developer mode
+
+2. Load unpacked extensions, select the extension directory in our project and click Reload
+
+3. Go to any job page, click the extension and you should see the following in chrome console:
```
Comment (chadwpry) [inline README.md]: It probably makes more sense to point to a specific url so you can be assured it has a selector.

**Comment by chadwpry** [comment]
Other than the comments I mentioned, that is all the feedback I have.

**Comment by robskrob** [comment]
@chadwpry thank you for this feedback. I updated my work according to your comments. Feel free to review whenever you can.

**Comment by chadwpry** [comment]
Let me know when you are finished with changes and I'll merge.

**Comment by robskrob** [comment]
I've submitted all of my changes.

File: extension/README.md [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+
+4. Create webpack bundle
+
+    `npm run build`
+
+5. Go to chrome://extensions/ in your browser and click developer mode
+
+6. Load unpacked extensions, select the extension directory in our project and click Reload
+
+7. Go to any job page, click the extension and you should see the following in chrome console:
```
Comment (chadwpry) [inline extension/README.md]: Update this from "any job" to a specific url that returns the example response below.

**Comment by robskrob** [comment]
Let me know if the URL is too specific. If the job is taken down, then one won't be able to reproduce this exact example.

**Comment by chadwpry** [comment]
I think it is alright. If it is down, someone can use another url on jobs.lever.co

**Pull request merged**